### DISCUSSION TITLE
部署在服务器上出码异常

### DISCUSSION THREAD

**Discussion by leeron-obj** [opentiny/tiny-engine#437] [Q&A]
![image](https://github.com/opentiny/tiny-engine/assets/114710659/490504b6-50d7-4e25-8dcd-6f013f5e5cbb)
怎么将generate-vue/src/http.js中的fetchCode改为调用vue-generator/src/generator/page中的generateCode方法出码

**Comment by chilingling** [comment]
1. 新版出码（预计5月份合入），会使用 vue-generator 的新版出码方法。
2. 如果需要改造当前的 vue-generator 在浏览器中调用，需要解决以下相关问题：
	1. vue-generator package 打包增加 es 格式的打包
	2. 在浏览器调用 vue-generator 的 generateCode 方法前，自行向服务端请求相关的页面数据（页面 schema）、页面数据如果使用了区块，需要级联请求区块 schema 数据，一并出码。

@leeron-obj

**Comment by chilingling** [comment]
当前新版出码已重构成直接调用  vue-generator package 的出码方法，欢迎体验
@leeron-obj
